### PR TITLE
Onboard centraluseuap to canary, onboard westcentralus to pilot

### PR DESCRIPTION
**Pull request by cadenmarchese**
### Which issue this PR addresses:

Partially fixes https://msazure.visualstudio.com/AzureRedHatOpenShift/_workitems/edit/13111297

### What this PR does / why we need it:

We are introducing Central US EUAP into canary - this PR includes the region in pipeline manifests alongside our other canary regions.

### Test plan for issue:

No

### Is there any documentation that needs to be updated for this PR?

No

### CODE REVIEW

**Review by s-amann — APPROVED** [review]

**Review by bennerv — CHANGES_REQUESTED** [review]
According to [SDP](https://eng.ms/docs/quality/zero-self-inflicted-sev1s/safedeploy), the rollout order should be canary regions (eastus2euap, centraluseuap), westcentralus, medium load regions, heavy load regions, primary regions, secondary regions.  Can we update the ordering to reflect this now in this PR as well?  

Should just need a reordering

**Comment by cadenmarchese** [comment]
@bennerv thanks, let me know if that works.

File: .pipelines/prod-release-tag.yml [hunk 1]
```diff
@@ -22,6 +22,7 @@ stages:
       locations:
       - westcentralus
```
Comment (bryanro92) [inline .pipelines/prod-release-tag.yml:23]: Please move `westcentralus` to its own sector named `pilot sector`
```
 jobs:
  - template: ./templates/template-job-deploy-azure-env-tag.yml
    parameters:
      environment: RP-Prod-PilotSector
      rpMode: ''
      aroVersionStorageAccount: $(aro-version-storage-account)
      locations:
      - westcentralus
      - eastus2euap
      - centraluseuap
      configFileName: prod-config.yaml
      azureDevOpsJSONSPN: $(aro-v4-ci-devops-spn)
      vsoProjectID: $(vso-project-id)
      azureDevOpsE2EJSONSPN: $(aro-v4-e2e-devops-spn)
      e2eSubscription: $(e2e-subscription)
      billingE2EPipelineName: $(billing-e2e-pipeline-name)
      billingE2EBranchName: $(billing-e2e-branch-name)
      imageTag: ${{ parameters.aroImageTag }}
      rpImageAcr: ${{ parameters.rpImageAcr }}
      ```

File: .pipelines/prod-release-tag.yml [hunk 2]
```diff
@@ -32,9 +31,29 @@ stages:
       billingE2EBranchName: $(billing-e2e-branch-name)
       imageTag: ${{ parameters.aroImageTag }}
       rpImageAcr: ${{ parameters.rpImageAcr }}
+- stage: Deploy_PilotSector
+  dependsOn: [Deploy_CanarySector]
+  displayName: 🚀 Deploy Pilot Sector
```
Comment (s-amann) [inline .pipelines/prod-release-tag.yml:36]: nit: can we throw in 👨‍✈️ / 👩‍✈️ in addition to or in replace of the rocket?

**Review by s-amann — CHANGES_REQUESTED** [review]
Please double check the non tagged release yaml and make sure we want/don't want to use the tagged template there.

File: .pipelines/prod-release.yml [hunk 3]
```diff
@@ -30,9 +29,29 @@ stages:
       e2eSubscription: $(e2e-subscription)
       billingE2EPipelineName: $(billing-e2e-pipeline-name)
       billingE2EBranchName: $(billing-e2e-branch-name)
+- stage: Deploy_PilotSector
+  dependsOn: [Deploy_CanarySector]
+  displayName: 🚀 Deploy Pilot Sector
+  jobs:
+  - template: ./templates/template-job-deploy-azure-env-tag.yml
```
[the remark this answers is not in the document]
Comment (cadenmarchese) [reply .pipelines/prod-release.yml]: Thanks, it was a bad copy/paste.

File: .pipelines/prod-release-tag.yml (hunk 2 above)
Comment (cadenmarchese) [reply .pipelines/prod-release-tag.yml]: Hmm, the py linter doesn't like my emoji extension. I'll try another.

**Review by s-amann — DISMISSED** [review]

**Review by bryanro92 — COMMENTED** [review]
this lgtm. I think we could go thru a "deploy all" thru canary with this to test it out and bail out before pilot region

**Comment by jhoreman** [comment]
What is the significance/implication of `westcentralus` being a "Pilot Sector" and not a canary? Will this require a change in our deployment/validation strategies? Code looks fine to me, just some clarification on pilot vs canary.

**Pull request closed without merging**